supabase/migrations/soft_delete_flashcard_sets.sql

```diff
@@ -0,0 +1,59 @@
+-- Add is_deleted column to public.flashcard_sets
+ALTER TABLE
+  public.flashcard_sets
+ADD
+  COLUMN IF NOT EXISTS is_deleted BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
We don't need  `is_deleted` column because if `deleted_at is not null`, it means this row was deleted.